packages.json

```diff
@@ -1636,6 +1636,11 @@
         "url": "https://github.com/dvkch/SYConfigurableVariable",
         "description": "Create build variables that you will be able to edit easily via the Edit menu in Xcode",
         "screenshot": "https://raw.githubusercontent.com/dvkch/SYConfigurableVariable/master/Sample%20screenshot%202.png"
+      },
+      {
+        "name": "Insert-Lines",
+        "url": "https://github.com/matrinox/InsertLineXcodePlugin",
+        "description": "Covenient inserting lines like in Sublime/Atom. Use command+enter or command+shift+enter to insert and jump to the next or previous line, respectively."
```

Comment:
It looks like you've listed this as a file template, but I think this is a plugin? You should add it under the `plugins` key, in that case :)